kafkazk/zookeeper_stub.go

```diff
@@ -1,3 +1,6 @@
+// This file is entirely for tests, but isn't defined as a _test file due to
```

Comment:
I don't know the implications of this

Reply:
Code in `<name>_test.go` files is exclusively used for testing; it's usually not visible in non-testing code or Godoc. In this case, we use some of the types/functions in here for tests in other packages, so it's setup like regular code (not a test file) so it's visible elsewhere.